tools/tf2openapi/types/tfmetagraph.go

```diff
@@ -6,31 +6,28 @@ It is the internal model representation for the MetaGraph defined in the TensorF
 [tensorflow/core/protobuf/meta_graph.proto]
 */
 import (
+	"fmt"
 	"github.com/getkin/kin-openapi/openapi3"
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
 )
 
-const PredictReqSigDefMethod string = "tensorflow/serving/predict"
-
 type TFMetaGraph struct {
-	SignatureDefs [] TFSignatureDef
+	SignatureDef TFSignatureDef
```

Comment:
So this is no longer a slice? Was the OR semantic too complicated?

Reply:
Yup, each SavedModel now only holds 1 MetaGraph, which holds 1 SignatureDef. I think it's better to generate OpenAPI specs for separate SignatureDefs separately because 
1. In TFServing, when serving a model, you can only specify one SignatureDef*. The OR semantic makes it seem like either one of the SignatureDefs can be used at a given time, which isn't the case.
2. Converting all SignatureDefs to internal model and to OpenAPI is inefficient when the user only needs one.

*https://github.com/tensorflow/serving/blob/97687024c3b7515d2f2979c35054f44c8f84d146/tensorflow_serving/model_servers/http_rest_api_handler.cc (line 313)

Reply:
I'm convinced!